app/views/api/v1/accounts/conversations/attachments.json.jbuilder

```diff
@@ -1 +1,10 @@
-json.payload @attachments.map(&:push_event_data)
+json.payload @attachments do |attachment|
+  json.message_id attachment.push_event_data[:message_id]
+  json.thumb_url attachment.push_event_data[:thumb_url]
+  json.data_url attachment.push_event_data[:data_url]
+  json.file_size attachment.push_event_data[:file_size]
+  json.file_type attachment.push_event_data[:file_type]
+  json.extension attachment.push_event_data[:extension]
+  json.created_at attachment.message.created_at.to_i
+  json.sender attachment.message.sender.push_event_data if attachment.message.sender
```

Comment:
Can we include this in the `push_event_data` directly? Thoughts @sojan-official?

Reply:
think we are using `attachment.push_event_data` in the message builder. could end up creating a cyclic dependency.  for now, let's go ahead with this.